### DISCUSSION TITLE
API for access controller (not images)

### DISCUSSION THREAD

**Discussion by kcan987** [AxisCommunications/discussions#187] [VAPIX] [answered]
We are searching for data about how to integrate any of your network door controllers into our software. like the A1001, A1210 or A1601. I searched the API documentation and it all appears to be related to Video camera access. We only need the ability to ADD new access PIN numbers and then ENABLE or DISABLE them all automatically from our software.

Can you direct me to any information about how to accomplish this with your product?
I did find this, https://www.axis.com/vapix-library/subjects/t10067552/section/t10068774/display?section=t10068774-t10069416
BUT I still have a questions about starting a billing account for the permission to use the Axis API 

I asked the regular helpdesk about opening an account for API access and they couldn't help.

**Comment by StatusAuto** [comment]
To add to this, we also have been looking at integrating the network door controllers, but have found that the WSDL links for the VAPIX Access Control library are dead (on the Introductions Page under Prerequisites).

Does anyone know where to get the WSDLs to import into C# classes?

Thanks.

**Comment by onie5** [comment]
VAPIX feature is not available on the newer door controllers, e.g. A1601, A1610, A1210, etc. This is to enable the Secure Entry feature for Axis Camera Station. The A1001 is still capable of VAPIX though.

**Reply by kcan987** [reply]
any idea what controller would be compatible with an API for integration into a custom software development?

**Reply by LeonardoNevesDuarte** [reply]
I have the same question... I've integrated our legacy system to A1001 and A1601 (with old firmware) controllers using the APIs but I just received a new A1610 access controller that is not supporting it... Do you guys know what is the recommended way to integrate other systems to these new models without using the APIs? I've seen there's a MQTT support but is this the new way?

**Reply by onie5** [reply, marked as answer]
Hi,
Just to let you know that a new firmware has been released to bring back the VAPIX functionality on the A16-series.

Please check out the corresponding product support pages.
